source/write_docs/content/index.html.md.erb

```diff
@@ -108,36 +65,64 @@ The following code is an example of a 2 column 2 row table with no text wrapping
 </div>
 ```
 
-## Links
+## Create a link
 
-```bash
-[LINK TEXT](LINK ADDRESS)
+You can create links using either markdown or HTML.
+
+To create a link in markdown, use the following:
+
+```yaml
+[link text](link address)
 ```
 
 For example:
 
-```bash
+```yaml
 [The Google home page](http://www.google.co.uk)
 ```
 
+To create a link in HTML, use the following:
+
+```bash
+<a href="link address">link text</a>
+```
+
 You can write the link in HTML with a `target="blank"` element to make the link open in a new internet browser tab by default:
 
 ```bash
-<a href="LINK ADDRESS" target="blank">LINK TEXT</a>
+<a href="link address" target="blank">link text</a>
 ```
 
-## Images
+## Insert an image
+
+To insert an image into your content, use the following:
 
-`!``[]``(LOCATION_OF_IMAGE)`
+```bash
+<%= image_tag "image location", { :alt => 'image alt text' } %>
+```
+
+You must include alt text because of accessibility requirements.
```

Comment:
Couple of things here:
- Is alt text definitely a requirement if the image has been explained in surrounding content?
- Middleman has updated how it generates alt text, so I think the `! [] ()` way of adding an image may work ok now?

Reply:
Also do you have to put images in the `images` folder or is that just a suggestion? (Also 'imagelocation' sounds a little like a folder - should it be image-file?)